packages/sil-voting-app/src/routes/dashboard/components/Proposal.tsx

```diff
@@ -0,0 +1,191 @@
+import Button from '@material-ui/core/Button';
+import Collapse from '@material-ui/core/Collapse';
+import Block from 'medulas-react-components/lib/components/Block';
+import Img from 'medulas-react-components/lib/components/Image';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React, { useState } from 'react';
+import deleteIcon from '../../../assets/delete.svg';
+import { elipsify } from '../../../utils/strings';
+
+const TITLE_MAX_LENGTH = 30;
+const DESC_MAX_LENGTH = 180;
+
+export interface VoteResult {
+  readonly yes: number;
+  readonly no: number;
+  readonly abstain: number;
+}
+
+interface VoteWinner {
+  readonly vote: string;
+  readonly percentage: number;
+}
+export interface ProposalProps {
+  readonly id: string;
+  readonly title: string;
+  readonly author: string;
+  readonly description: string;
+  readonly creationDate: Date;
+  readonly expiryDate: Date;
+  readonly quorum: number;
+  readonly threshold: number;
+  readonly result: VoteResult;
+  readonly vote: 'Invalid' | 'Yes' | 'No' | 'Abstain';
+  readonly status: 'Active' | 'Submitted' | 'Ended';
+}
+
+const getTotalVotes = (result: VoteResult): number => {
+  return result.yes + result.no + result.abstain;
+};
+
+const getWinner = (result: VoteResult): VoteWinner => {
+  const maxVotes = Math.max(result.yes, result.no);
+  let vote = 'Abstain';
+
+  if (maxVotes === result.yes) vote = 'Yes';
+  if (maxVotes === result.no) vote = 'No';
+  if (maxVotes === 0) vote = 'Abstain';
+
+  const percentage = Math.floor((maxVotes / getTotalVotes(result)) * 100);
+
+  return { vote, percentage };
+};
+
+const Proposal = (props: ProposalProps): JSX.Element => {
+  const [expanded, setExpanded] = useState(false);
+
+  const onClick = (): void => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <Block width="100%" display="flex" alignItems="center">
+      <Block flexGrow={1} margin={2}>
+        <Block display="flex" alignItems="center">
+          <Typography variant="h6">{elipsify(props.title, TITLE_MAX_LENGTH)}</Typography>
+          <Block marginLeft={2}>
+            <Typography variant="body1">{props.status}</Typography>
+          </Block>
+        </Block>
+        <Block display="flex" alignItems="center">
+          <Typography variant="body1">Author: {props.author}</Typography>
+          <Block marginLeft={2}>
+            <Typography variant="body1">Proposal ID: {props.id}</Typography>
+          </Block>
+          <Block marginLeft={2}>
+            <Typography variant="body1">
+              Created on {props.creationDate.toLocaleDateString('es-ES')}
+            </Typography>
+          </Block>
+        </Block>
+        {props.description.length < DESC_MAX_LENGTH && (
```

Comment:
From line 81 to 105 should be extracted into:
`src/routes/dashboard/components/Proposal/Description.tsx` note Proposal.tsx is now -> `dashboard/components/Proposal/intex.tsx`